apps/paradym/package.json

```diff
@@ -10,6 +10,7 @@
 		"prebuild": "APP_VARIANT=development pnpm expo prebuild --no-install"
 	},
 	"dependencies": {
+		"@animo-id/expo-secure-environment": "*",
```

Comment:
I needed to add this to start the wallet, not sure why it's needed

Reply:
What was the error you got otherwise?